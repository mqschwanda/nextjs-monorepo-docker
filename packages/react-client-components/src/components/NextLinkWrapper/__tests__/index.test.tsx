import { render } from '@testing-library/react';
import { NextLinkWrapper } from '@';

describe('@mqs/react-client-components', () => {
  describe('components', () => {
    describe('<NextLinkWrapper />', () => {
      it('renders', () => {
        const testId = NextLinkWrapper.name;
        const chilren = 'children...';
        const href = '/home';

        const { getByTestId, asFragment } = render((
          <NextLinkWrapper
            href={href}
            testId={testId}
          >
            <a
              href={href}
            >
              { chilren }
            </a>
          </NextLinkWrapper>
        ));

        const element = getByTestId(testId);
        expect(element).toBeTruthy();

        expect(asFragment()).toMatchSnapshot();
      });
    });
  });
});
