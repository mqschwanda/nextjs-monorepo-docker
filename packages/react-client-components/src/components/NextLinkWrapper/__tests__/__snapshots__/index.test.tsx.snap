// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@mqs/react-client-components components <NextLinkWrapper /> renders 1`] = `
<DocumentFragment>
  <a
    data-testid="NextLinkWrapper"
    href="/home"
  >
    children...
  </a>
</DocumentFragment>
`;
